lib/gems/pending/appliance_console/cli.rb

```diff
@@ -214,6 +226,22 @@ def set_external_db
       config.post_activation
     end
 
+    def set_date_time
+      date = options[:date]
+      time = options[:time]
+      if date =~ ApplianceConsole::DateTimeConfiguration::DATE_REGEXP &&
+         time =~ ApplianceConsole::DateTimeConfiguration::TIME_REGEXP
+        say "Configuring date and time"
+        @new_date = date
+        @new_time = time
+        @manual_time_sync = options[:timesync]
```

Comment:
Why are these instance variables?

I think these should probably be setting the instance variables of `config_datetime` below.
As of now it looks like this would unconditionally set the time sync to manual and then fail to set the date and time as those fields in the `DateTimeConfiguration` object would be `nil`.